src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    // operate : (calback, arg) => {
+    //   calback(arg);
+    // },
+  };
+
+  calculator['operate'] = (calback, arg) => {
+    calback(arg);
+
+    return calculator;
+  };
+
+  calculator['add'] = (arg) => {
+    calculator.result += arg;
```

Comment:
better to use `this` instead of calculator, it will improve the reusability

Reply:
I've just push commit 'second implementation'. Please check it. But I can't use 'this` instead calculator[property](